tests/integration/allocator_integration_tests.cpp

```diff
@@ -39,7 +39,8 @@ TEST_P(AllocatorTest, AllocateDeallocateBig)
       static_cast<double*>(m_allocator->allocate(m_big * sizeof(double)));
 
   ASSERT_NE(nullptr, data);
-
+  ASSERT_EQ(nullptr, m_allocator->getParent());
+  
```

Comment:
Is there any way we can extend this test to also confirm we got the parent we are expecting by examining the traits of the parent?

Reply:
Also, how do the zero-byte allocators and introspection allocators fit in to the parent/child scheme?  Are they ever visible to the Umpire user?